exercises/practice/roman-numerals/.meta/example.ua

```diff
@@ -0,0 +1,7 @@
+Hundreds      ← {"" "C" "CC" "CCC" "CD" "D" "DC" "DCC" "DCCC" "CM"}
+Tens          ← {"" "X" "XX" "XXX" "XL" "L" "LX" "LXX" "LXXX" "XC"}
+Ones          ← {"" "I" "II" "III" "IV" "V" "VI" "VII" "VIII" "IX"}
+Lookup        ← comptime(☇ 0 ≡≡(⍚/◇⊂) ⊞(⊂⊂) Hundreds Tens Ones)
```

Comment:
I did not expect compile time support, but it is supported!

Reply:
I saw that but had no inspiration what to do with it. Now I've got to look it up closely

Reply:
Did you mean not to approce this PR?

Reply:
I meant the `comptime` function

Reply:
`Lookup ← ♭ ⊞⍚(⊂⊂) Hundreds Tens Ones`